compass_app/app/lib/ui/activities/view_models/activities_viewmodel.dart

```diff
@@ -13,70 +14,102 @@ class ActivitiesViewModel extends ChangeNotifier {
   })  : _activityRepository = activityRepository,
         _itineraryConfigRepository = itineraryConfigRepository {
     loadActivities = Command0(_loadActivities)..execute();
+    saveActivities = Command0(() async {
+      _log.shout(
+        'Save activities not implemented',
+        null,
+        StackTrace.current,
+      );
+      return Result.error(Exception('Not implemented'));
+    });
   }
 
+  final _log = Logger('ActivitiesViewModel');
   final ActivityRepository _activityRepository;
   final ItineraryConfigRepository _itineraryConfigRepository;
-  List<Activity> _activities = <Activity>[];
+  List<Activity> _daytimeActivities = <Activity>[];
+  List<Activity> _eveningActivities = <Activity>[];
   final Set<String> _selectedActivities = <String>{};
 
-  /// List of [Activity] per destination.
-  List<Activity> get activities => _activities;
+  /// List of daytime [Activity] per destination.
+  List<Activity> get daytimeActivities => _daytimeActivities;
+
+  /// List of evening [Activity] per destination.
+  List<Activity> get eveningActivities => _eveningActivities;
```

Comment:
nit / nonblocking comment:
I've recently learned about [UnmodifiableListView](https://api.dart.dev/stable/3.4.4/dart-collection/UnmodifiableListView-class.html), and I think this is the perfect place to use it :)

```
  /// List of daytime [Activity] per destination.
  UnmodifiableListView<Activity> get daytimeActivities => UnmodifiableListView(_daytimeActivities);
```

Reply:
I will give it a try!